common/v2/config/fiats.ts

```diff
@@ -13,12 +13,38 @@ export const USD = {
 export const EUR = {
   code: 'EUR',
   name: 'Euros',
-  symbol: '€' as TSymbol
+  symbol: '€' as TSymbol,
+  prefix: true
 };
 export const GBP = {
   code: 'GBP',
   name: 'British Pounds',
-  symbol: '£' as TSymbol
+  symbol: '£' as TSymbol,
+  prefix: true
```

Comment:
Not sure if prefixing by default is the best option? I think I'm used to seeing it suffixed unless it's USD?

Reply:
It seems to be based on locality of the user, not on the currency symbol itself. But if it's not actually a symbol, but a string of letters instead you're supposed to post-fix. We don't really have a way to implement the localities though.. soooo?